backend/core/src/listings/types/listing-filter-keys-enum.ts

```diff
@@ -10,6 +10,7 @@ export enum ListingFilterKeys {
   maxRent = "maxRent",
   ami = "ami",
   leasingAgents = "leasingAgents",
+  include_nulls = "include_nulls",
```

Comment:
I need to think about this more, but just to so you know I looked at it:
This is necessary because of how we're relying on the ListingFilterParams as the storage object for the frontend state, but i don't think we should do that anymore.

Elen's PR #484 fixes this.

Reply:
I think it will look mostly similar, but `includeNulls` would be added to `FrontendFilterKeys`, and a corresponding entry set in `FrontendFilterState` ctor.
Is there an eta on #484 ? I'd say this PR should wait until that one is submitted, unless it's not close to being ready.

Reply:
Instead of adding a key here, we should add it to `FrontendListingFilterStateKeys` on line 46 of `filters.ts`, with something like
```
export const FrontendListingFilterStateKeys = { ...ListingFilterKeys, "includeNulls" }
```

The reason is that `ListingFilterKeys` is meant to be a list of the available filters on the backend, and "includeNulls" isn't itself a filter, but just a modifier of other filters.